content/zh/docs/concepts/services-networking/service-topology.md

```diff
@@ -1,35 +1,42 @@
 ---
+title: Service Topology
+weight: 20
+date: 2020-09-09
+publishdate: 2020-09-09
+content_type: concept
+---
+
+<!--
+---
 reviewers:
 - johnbelamaric
 - imroc
-title: Service 拓扑
+title: Service Topology
 feature:
-  title: Service 拓扑
+  title: Service Topology
   description: >
-    基于集群拓扑的服务流量路由。
+    Routing of service traffic based upon cluster topology.
 
 content_type: concept
 weight: 10
 ---
-
+ -->
 
 <!-- overview -->
-
+<!--
 {{< feature-state for_k8s_version="v1.17" state="alpha" >}}
```

Comment:
Don't wrap any shortcode inside HTML comments because the shortcode will still work.